examples/object_detection_2d/object_detection_2d/seed_test_suite.py

```diff
@@ -0,0 +1,156 @@
+
+            # check that the box is in a valid image, not a crowd box, and in the transportation supercategory
+            if image_id in ids and not annotation["iscrowd"] and label in TRANSPORTATION_LABELS:
+                bbox = annotation["bbox"]
+                top_left = (bbox[0], bbox[1])
+                bottom_right = (bbox[0] + bbox[2], bbox[1] + bbox[3])
+                bounding_box = LabeledBoundingBox(top_left, bottom_right, label)
+                image_to_boxes["COCO_val2014_" + str(image_id).zfill(12) + ".jpg"].append(bounding_box)
+        return image_to_boxes
+
+
+def create_complete_transportation_case(args: Namespace) -> TestCase:
+    image_to_boxes = load_transportation_data()
+    data = pd.read_csv(args.metadata)
+
+    # create a test sample object and a ground truth object per image
+    test_samples_and_ground_truths: List[Tuple[TestSample, GroundTruth]] = []
+    for _, row in data.iterrows():
+        image_name = row["name"]
+        test_sample = TestSample(
+            locator=S3_IMAGE_LOCATION + image_name,
+            metadata={
+                "image_width": row["width"],
+                "image_height": row["height"],
+                "brightness": row["brightness"],
+            },
+        )
+
+        ground_truth = GroundTruth(
+            bboxes=image_to_boxes[image_name],
+            ignored_bboxes=[],
+        )
+
+        test_samples_and_ground_truths.append((test_sample, ground_truth))
+
+    # create the complete test case, not attached to any test suite
+    complete_test_case = TestCase(
+        f"complete transportation :: {DATASET}",
+        test_samples=test_samples_and_ground_truths,
+        reset=True,
+    )
+
+    return complete_test_case
+
+
+def seed_test_suite_by_brightness(test_suite_name: str, complete_test_case: TestCase) -> None:
+    stratification_logic_map = {
+        "light": lambda brightness: brightness >= 140,
+        "normal": lambda brightness: 90 <= brightness < 140,
+        "dark": lambda brightness: 0 <= brightness < 90,
+    }
+
+    # create each test case by stratification
+    test_cases: List[TestCase] = []
+    for name, fn in stratification_logic_map.items():
+        filtered_test_samples = [
+            (ts, gt) for ts, gt in complete_test_case.iter_test_samples() if fn(ts.metadata["brightness"])
+        ]
+        new_test_case = TestCase(
+            f"brightness :: {name} :: {DATASET}",
+            test_samples=filtered_test_samples,
+            reset=True,
+        )
+        test_cases.append(new_test_case)
+
+    # create the test suite with the complete test case and new test cases
+    test_suite = TestSuite(
+        test_suite_name,
+        description=f"{TEST_SUITE_DESCRIPTION}, stratified by `light`, `normal`, and `dark` brightness",
+        test_cases=[complete_test_case, *test_cases],
+        reset=True,
+    )
+
+    print(f"created test suite {test_suite.name} v{test_suite.version}")
```

Comment:
Nit: wrap `test_suite.name` in single quotes so it's easier to copy/paste to `seed_test_run` command